docs/contributing/testing.md

```diff
@@ -4,20 +4,25 @@ title: Contributing
 sidebar_label: Testing
 ---
 
-# Testing
+---
```

Comment:
This page is now broken.  Please follow these instructions to test out locally the docusaurus generated content,.
```bash
cd /website
npm run start
```
then visit http://localhost:3000/fast-dna/docs/contributing/testing after the web server starts up.

Reply:
![screen shot 2018-12-23 at 10 02 42 am](https://user-images.githubusercontent.com/16669785/50385865-ea2b9b00-0699-11e9-8184-ee326fc68b01.png)

Reply:
I think it's just because there is a duplicate header in the file.

Reply:
Also, once this PR goes in let me know and I can run the publish to update the actual GitHub Pages or you can try if you want.  https://docusaurus.io/docs/en/publishing

Reply:
Had to npm install first.
Then I did a build and start.
Found another issue where each page had a Contributions title, removed the extra title.
All changes validated on my machine.